bftengine/src/bcstatetransfer/BCStateTran.cpp

```diff
@@ -2502,6 +2499,14 @@ void BCStateTran::SetAggregator(std::shared_ptr<concordMetrics::Aggregator> aggr
   metrics_component_.SetAggregator(aggregator);
 }
 
+inline std::string BCStateTran::getSequenceNumber(uint16_t replicaId,
+                                                  uint64_t seqNum,
+                                                  uint16_t blockNum,
+                                                  uint64_t chunkNum) {
```

Comment:
What happens if those two aren't provided? Shouldn't they have default values?

Reply:
They have default values:
inline std::string getSequenceNumber(uint16_t replicaId, uint64_t seqNum, uint16_t = 0, uint64_t = 0);

Reply:
thanks for the clarification!